roachpb/internal.proto

```diff
@@ -75,12 +73,13 @@ message InternalTimeSeriesSample {
   // the TimeSeriesData collection.
   optional int32 offset = 1 [(gogoproto.nullable) = false];
 
-  // Count of measurements taken within this sample.
-  optional uint32 count = 6 [(gogoproto.nullable) = false];
-  // Sum of all measurements.
-  optional double sum = 7 [(gogoproto.nullable) = false];
+  // A series of measurements gathered over a time period.
+  repeated double value = 2;
   // Maximum encountered measurement in this sample.
   optional double max = 8;
   // Minimum encountered measurement in this sample.
   optional double min = 9;
+
+  optional uint32 DELETED_count = 6 [(gogoproto.nullable) = false];
+  optional double DELETED_sum = 7 [(gogoproto.nullable) = false];
```

Comment:
I'm not sure you can mark these fields as `DELETED` just yet. Won't new servers need to read these fields if they exist?